Subscription details change owner bug

**Version Info**
v18.1 master

**Describe the bug**
User is unable to change subscription owner, the modal dialog is not appearing on the screen.

**How to Reproduce**
Steps to reproduce the behavior:
1. Subscribe to the service
2. Activate the service
3. Go to oscm-portal/marketplace/account/subscriptions.jsf
4. Select subscription details
5. Go to configure tab
6. Click the pencil next to owner label, a modal dialog is not appearing.

**Observed behavior**
After clicking the pencil at the configuration tab, the modal to change the owner is not appearing. 

**Expected behavior**
After clicking the pencil at the configuration tab, the modal to change the owner should appear. 

**Screenshots**
N/A

**Additional context**
An exception is thrown inside the logs.
```
05/27/2020_10:29:01.036 FSP_INTS-BSS: ERROR: ThreadID https-jsse-nio-8081-exec-8: ExceptionActionListener: ?? key 'ERROR' not found ??
javax.faces.FacesException: #{manageSubscriptionCtrl.initializeSubscriptionOwners}: javax.ejb.EJBException: The bean encountered a non-application exception; nested exception is:
        javax.ejb.EJBAccessException: Illegal access from principal 1000
        at com.sun.faces.application.ActionListenerImpl.processAction(ActionListenerImpl.java:118) ~[javax.faces-2.2.12.jar:2.2.12]
        at org.oscm.ui.common.ExceptionActionListener.processAction(ExceptionActionListener.java:41) [classes/:PRIVATE-BUILD-ON-G02PLXN03045-20200527101041]
        at org.omnifaces.eventlistener.ResetInputAjaxActionListener.processAction(ResetInputAjaxActionListener.java:197) [omnifaces-1.10.jar:1.10]
        at javax.faces.component.UICommand.broadcast(UICommand.java:315) [javax.faces-2.2.12.jar:2.2.12]
        at javax.faces.component.UIViewRoot.broadcastEvents(UIViewRoot.java:790) [javax.faces-2.2.12.jar:2.2.12]
        at javax.faces.component.UIViewRoot.processApplication(UIViewRoot.java:1282) [javax.faces-2.2.12.jar:2.2.12]
        at com.sun.faces.lifecycle.InvokeApplicationPhase.execute(InvokeApplicationPhase.java:81) [javax.faces-2.2.12.jar:2.2.12]
        at com.sun.faces.lifecycle.Phase.doPhase(Phase.java:101) [javax.faces-2.2.12.jar:2.2.12]
        at com.sun.faces.lifecycle.LifecycleImpl.execute(LifecycleImpl.java:198) [javax.faces-2.2.12.jar:2.2.12]
        at javax.faces.webapp.FacesServlet.service(FacesServlet.java:658) [javax.faces-2.2.12.jar:2.2.12]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.apache.myfaces.webapp.filter.ExtensionsFilter.doFilter(ExtensionsFilter.java:357) [tomahawk20-1.1.14.jar:1.1.14]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.AuthorizationFilter.handleLoggedInUser(AuthorizationFilter.java:742) [classes/:PRIVATE-BUILD-ON-G02PLXN03045-20200527101041]
        at org.oscm.ui.filter.AuthorizationFilter.handleProtectedUrlAndChangePwdCase(AuthorizationFilter.java:304) [classes/:PRIVATE-BUILD-ON-G02PLXN03045-20200527101041]
        at org.oscm.ui.filter.AuthorizationFilter.doFilter(AuthorizationFilter.java:132) [classes/:PRIVATE-BUILD-ON-G02PLXN03045-20200527101041]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.NonexistentConversationFilter.doFilter(NonexistentConversationFilter.java:37) [classes/:PRIVATE-BUILD-ON-G02PLXN03045-20200527101041]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.MarketplaceContextFilter.doFilter(MarketplaceContextFilter.java:112) [classes/:PRIVATE-BUILD-ON-G02PLXN03045-20200527101041]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.apache.openejb.server.httpd.EEFilter.doFilter(EEFilter.java:65) [openejb-http-7.1.1.jar:7.1.1]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.DisableUrlFilter.doFilter(DisableUrlFilter.java:52) [classes/:PRIVATE-BUILD-ON-G02PLXN03045-20200527101041]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.RoleBasedFilter.doFilter(RoleBasedFilter.java:82) [classes/:PRIVATE-BUILD-ON-G02PLXN03045-20200527101041]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.ClosedMarketplaceFilter.doFilter(ClosedMarketplaceFilter.java:112) [classes/:PRIVATE-BUILD-ON-G02PLXN03045-20200527101041]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.IllegalRequestParameterFilter.doFilter(IllegalRequestParameterFilter.java:81) [classes/:PRIVATE-BUILD-ON-G02PLXN03045-20200527101041]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.OidcFilter.doFilter(OidcFilter.java:126) [classes/:PRIVATE-BUILD-ON-G02PLXN03045-20200527101041]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.OidcLogoutFilter.doFilter(OidcLogoutFilter.java:90) [classes/:PRIVATE-BUILD-ON-G02PLXN03045-20200527101041]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.HttpMethodFilter.doFilter(HttpMethodFilter.java:69) [classes/:PRIVATE-BUILD-ON-G02PLXN03045-20200527101041]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.DisableUrlSessionFilter.doFilter(DisableUrlSessionFilter.java:50) [classes/:PRIVATE-BUILD-ON-G02PLXN03045-20200527101041]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.CharacterEncodingFilter.doFilter(CharacterEncodingFilter.java:57) [classes/:PRIVATE-BUILD-ON-G02PLXN03045-20200527101041]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:199) [catalina.jar:8.5.41]
        at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96) [catalina.jar:8.5.41]
        at org.apache.tomee.catalina.OpenEJBValve.invoke(OpenEJBValve.java:44) [tomee-catalina-7.1.1.jar:7.1.1]
        at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:493) [catalina.jar:8.5.41]
        at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:137) [catalina.jar:8.5.41]
        at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81) [catalina.jar:8.5.41]
        at org.apache.tomee.catalina.OpenEJBSecurityListener$RequestCapturer.invoke(OpenEJBSecurityListener.java:97) [tomee-catalina-7.1.1.jar:7.1.1]
        at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:660) [catalina.jar:8.5.41]
        at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87) [catalina.jar:8.5.41]
        at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343) [catalina.jar:8.5.41]
        at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:798) [tomcat-coyote.jar:8.5.41]
        at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66) [tomcat-coyote.jar:8.5.41]
        at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:808) [tomcat-coyote.jar:8.5.41]
        at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1498) [tomcat-coyote.jar:8.5.41]
        at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) [tomcat-coyote.jar:8.5.41]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_252]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_252]
        at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-util.jar:8.5.41]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_252]
Caused by: javax.faces.el.EvaluationException: javax.ejb.EJBException: The bean encountered a non-application exception; nested exception is:
        javax.ejb.EJBAccessException: Illegal access from principal 1000
        at javax.faces.component.MethodBindingMethodExpressionAdapter.invoke(MethodBindingMethodExpressionAdapter.java:101) ~[javax.faces-2.2.12.jar:2.2.12]
        at com.sun.faces.application.ActionListenerImpl.processAction(ActionListenerImpl.java:102) ~[javax.faces-2.2.12.jar:2.2.12]
        ... 74 more
Caused by: javax.ejb.EJBException: The bean encountered a non-application exception; nested exception is:
        javax.ejb.EJBAccessException: Illegal access from principal 1000
        at org.apache.openejb.core.ivm.BaseEjbProxyHandler.convertException(BaseEjbProxyHandler.java:447) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.ivm.BaseEjbProxyHandler.invoke(BaseEjbProxyHandler.java:351) ~[openejb-core-7.1.1.jar:7.1.1]
        at com.sun.proxy.$Proxy427.getSubscriptionOwnersForAssignment(Unknown Source) ~[?:?]
        at org.oscm.ui.dialog.mp.subscriptionDetails.ManageSubscriptionCtrl.initializeSubscriptionOwners(ManageSubscriptionCtrl.java:699) ~[classes/:PRIVATE-BUILD-ON-G02PLXN03045-20200527101041]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_252]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_252]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_252]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_252]
        at org.apache.el.parser.AstValue.invoke(AstValue.java:247) ~[jasper-el.jar:8.5.41]
        at org.apache.el.MethodExpressionImpl.invoke(MethodExpressionImpl.java:267) ~[jasper-el.jar:8.5.41]
        at org.apache.webbeans.el22.WrappedMethodExpression.invoke(WrappedMethodExpression.java:52) ~[openwebbeans-el22-1.7.6.jar:1.7.6]
        at com.sun.faces.facelets.el.TagMethodExpression.invoke(TagMethodExpression.java:105) ~[javax.faces-2.2.12.jar:2.2.12]
        at javax.faces.component.MethodBindingMethodExpressionAdapter.invoke(MethodBindingMethodExpressionAdapter.java:87) ~[javax.faces-2.2.12.jar:2.2.12]
        at com.sun.faces.application.ActionListenerImpl.processAction(ActionListenerImpl.java:102) ~[javax.faces-2.2.12.jar:2.2.12]
        ... 74 more
Caused by: javax.ejb.EJBAccessException: Illegal access from principal 1000
        at org.oscm.operatorservice.bean.UserRolesInterceptor.throwIllegalAccess(UserRolesInterceptor.java:37) ~[?:?]
        at org.oscm.operatorservice.bean.UserRolesInterceptor.checkAccess(UserRolesInterceptor.java:30) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_252]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_252]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_252]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_252]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext$Invocation.invoke(ReflectionInvocationContext.java:205) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext.proceed(ReflectionInvocationContext.java:186) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.oscm.interceptor.ExceptionMapper.mapEJBExceptions(ExceptionMapper.java:49) ~[?:?]
        at sun.reflect.GeneratedMethodAccessor136.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_252]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_252]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext$Invocation.invoke(ReflectionInvocationContext.java:205) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext.proceed(ReflectionInvocationContext.java:186) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.oscm.interceptor.InvocationDateContainer.setTransactionTime(InvocationDateContainer.java:41) ~[?:?]
        at sun.reflect.GeneratedMethodAccessor222.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_252]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_252]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext$Invocation.invoke(ReflectionInvocationContext.java:205) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext.proceed(ReflectionInvocationContext.java:186) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.monitoring.StatsInterceptor.record(StatsInterceptor.java:191) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.monitoring.StatsInterceptor.invoke(StatsInterceptor.java:102) ~[openejb-core-7.1.1.jar:7.1.1]
        at sun.reflect.GeneratedMethodAccessor135.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_252]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_252]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext$Invocation.invoke(ReflectionInvocationContext.java:205) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext.proceed(ReflectionInvocationContext.java:186) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.interceptor.InterceptorStack.invoke(InterceptorStack.java:85) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.stateless.StatelessContainer._invoke(StatelessContainer.java:252) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.stateless.StatelessContainer.invoke(StatelessContainer.java:212) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.ivm.EjbObjectProxyHandler.synchronizedBusinessMethod(EjbObjectProxyHandler.java:265) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.ivm.EjbObjectProxyHandler.businessMethod(EjbObjectProxyHandler.java:260) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.ivm.EjbObjectProxyHandler._invoke(EjbObjectProxyHandler.java:89) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.ivm.BaseEjbProxyHandler.invoke(BaseEjbProxyHandler.java:347) ~[openejb-core-7.1.1.jar:7.1.1]
        at com.sun.proxy.$Proxy427.getSubscriptionOwnersForAssignment(Unknown Source) ~[?:?]
        at org.oscm.ui.dialog.mp.subscriptionDetails.ManageSubscriptionCtrl.initializeSubscriptionOwners(ManageSubscriptionCtrl.java:699) ~[classes/:PRIVATE-BUILD-ON-G02PLXN03045-20200527101041]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_252]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_252]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_252]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_252]
        at org.apache.el.parser.AstValue.invoke(AstValue.java:247) ~[jasper-el.jar:8.5.41]
        at org.apache.el.MethodExpressionImpl.invoke(MethodExpressionImpl.java:267) ~[jasper-el.jar:8.5.41]
        at org.apache.webbeans.el22.WrappedMethodExpression.invoke(WrappedMethodExpression.java:52) ~[openwebbeans-el22-1.7.6.jar:1.7.6]
        at com.sun.faces.facelets.el.TagMethodExpression.invoke(TagMethodExpression.java:105) ~[javax.faces-2.2.12.jar:2.2.12]
        at javax.faces.component.MethodBindingMethodExpressionAdapter.invoke(MethodBindingMethodExpressionAdapter.java:87) ~[javax.faces-2.2.12.jar:2.2.12]
        at com.sun.faces.application.ActionListenerImpl.processAction(ActionListenerImpl.java:102) ~[javax.faces-2.2.12.jar:2.2.12]
        ... 74 more
```

--- a/oscm-operatorsvc-unittests/javasrc/org/oscm/operatorservice/bean/UserRolesInterceptorTest.java
+++ b/oscm-operatorsvc-unittests/javasrc/org/oscm/operatorservice/bean/UserRolesInterceptorTest.java
@@ -1,0 +1,181 @@
+/**
+ * *****************************************************************************
+ *
+ * <p>Copyright FUJITSU LIMITED 2020
+ *
+ * <p>Creation Date: 03.06.2020
+ *
+ * <p>*****************************************************************************
+ */
+package org.oscm.operatorservice.bean;
+
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import java.lang.reflect.Method;
+
+import javax.ejb.EJBAccessException;
+import javax.ejb.SessionContext;
+import javax.interceptor.InvocationContext;
+
+import org.junit.Before;
+import org.junit.Test;
+
+/** @author goebel */
+public class UserRolesInterceptorTest {
+
+  private UserRolesInterceptor interceptor;
+  private InvocationContext invokeContext;
+
+  @Before
+  public void setup() throws Exception {
+    OperatorServiceBean bean = new OperatorServiceBean();
+    interceptor = new UserRolesInterceptor();
+    interceptor.ejbCtx = mock(SessionContext.class);
+    invokeContext = mock(InvocationContext.class);
+    doReturn(null).when(invokeContext).proceed();
+  }
+
+  @Test
+  public void checkAccess_getConfigurationSettings_Success() throws Exception {
+    // given
+    givenPlatformOperatorRole();
+    InvocationContext ic = givenInvokation("getConfigurationSettings");
+
+    // when
+    interceptor.checkAccess(ic);
+
+    // then
+    verify(interceptor.ejbCtx, times(1)).isCallerInRole("PLATFORM_OPERATOR");
+  }
+
+  @Test(expected = EJBAccessException.class)
+  public void checkAccess_getConfigurationSettings_NoOperatorRole() throws Exception {
+    // given
+    givenNoPlatformOperatorRole();
+    InvocationContext ic = givenInvokation("getConfigurationSettings");
+
+    // when
+    interceptor.checkAccess(ic);
+  }
+
+  @Test
+  public void checkAccess_getUnassignedUsersByOrg_Success() throws Exception {
+    // given
+    givenSubscriptionManagerRole();
+    givenUnitAdminRole();
+    givenOrgAdminRole();
+    InvocationContext ic = givenInvokation("getUnassignedUsersByOrg");
+
+    // when
+    interceptor.checkAccess(ic);
+
+    // then
+    verify(interceptor.ejbCtx, times(1)).isCallerInRole(anyString());
+  }
+
+  @Test
+  public void checkAccess_getUnassignedUsersByOrg_OneRole() throws Exception {
+    // given
+    givenUnitAdminRole();
+    InvocationContext ic = givenInvokation("getUnassignedUsersByOrg");
+
+    // when
+    interceptor.checkAccess(ic);
+
+    // then
+    verify(interceptor.ejbCtx, times(1)).isCallerInRole("UNIT_ADMINISTRATOR");
+  }
+
+  @Test(expected = EJBAccessException.class)
+  public void checkAccess_getConfigurationSettings_NoRole() throws Exception {
+    // given
+    givenNoRole();
+    InvocationContext ic = givenInvokation("getUnassignedUsersByOrg");
+
+    // when
+    interceptor.checkAccess(ic);
+  }
+
+  @Test
+  public void checkAccess_registerOrganization_Success() throws Exception {
+    // given
+    givenPlatformOperatorRole();
+    InvocationContext ic = givenInvokation("registerOrganization");
+
+    // when
+    interceptor.checkAccess(ic);
+
+    // then
+    verify(interceptor.ejbCtx, times(1)).isCallerInRole("PLATFORM_OPERATOR");
+  }
+
+  @Test(expected = EJBAccessException.class)
+  public void checkAccess_registerOrganization_NoRole() throws Exception {
+    // given
+    givenNoRole();
+    InvocationContext ic = givenInvokation("registerOrganization");
+
+    // when
+    interceptor.checkAccess(ic);
+  }
+
+  public void checkAccess_getOrganization_Success() throws Exception {
+    // given
+    givenNoRole();
+    InvocationContext ic = givenInvokation("getOrganization");
+
+    // when
+    interceptor.checkAccess(ic);
+
+    // then
+    verify(interceptor.ejbCtx, never()).isCallerInRole(anyString());
+  }
+
+  InvocationContext givenInvokation(String method) {
+    Method m = getMethod(method);
+    doReturn(m).when(invokeContext).getMethod();
+    return invokeContext;
+  }
+
+  void givenOrgAdminRole() {
+    doReturn(Boolean.TRUE).when(interceptor.ejbCtx).isCallerInRole("ORGANIZATION_ADMIN");
+  }
+
+  void givenUnitAdminRole() {
+    doReturn(Boolean.TRUE).when(interceptor.ejbCtx).isCallerInRole("UNIT_ADMINISTRATOR");
+  }
+
+  void givenSubscriptionManagerRole() {
+    doReturn(Boolean.TRUE).when(interceptor.ejbCtx).isCallerInRole("SUBSCRIPTION_MANAGER");
+  }
+
+  void givenNoSubscriptionManagerRole() {
+    doReturn(Boolean.FALSE).when(interceptor.ejbCtx).isCallerInRole("SUBSCRIPTION_MANAGER");
+  }
+
+  void givenPlatformOperatorRole() {
+    doReturn(Boolean.TRUE).when(interceptor.ejbCtx).isCallerInRole("PLATFORM_OPERATOR");
+  }
+
+  void givenNoPlatformOperatorRole() {
+    doReturn(Boolean.FALSE).when(interceptor.ejbCtx).isCallerInRole("PLATFORM_OPERATOR");
+  }
+
+  void givenNoRole() {
+    doReturn(Boolean.FALSE).when(interceptor.ejbCtx).isCallerInRole(anyString());
+  }
+
+  Method getMethod(String name) {
+    for (Method m : OperatorServiceBean.class.getDeclaredMethods()) {
+      if (m.getName().equals(name)) {
+        return m;
+      }
+    }
+    throw new RuntimeException(String.format("Methode %s Not found", name));
+  }
+}

--- a/oscm-operatorsvc/javasrc/org/oscm/operatorservice/bean/UserRolesInterceptor.java
+++ b/oscm-operatorsvc/javasrc/org/oscm/operatorservice/bean/UserRolesInterceptor.java
@@ -47,8 +47,11 @@ public class UserRolesInterceptor {
     RolesAllowed ra = context.getMethod().getDeclaredAnnotation(RolesAllowed.class);
     if (ra != null) {
       List<String> roles = Arrays.asList(ra.value());
-      for (String role : roles) {
-        if (!ejbCtx.isCallerInRole(role)) return false;
+      if (!roles.isEmpty()) {
+        for (String role : roles) {
+          if (ejbCtx.isCallerInRole(role)) return true;
+        }
+        return false;
       }
     }
     return true;

